comb_spec_searcher/comb_spec_searcher.py

```diff
@@ -878,8 +880,7 @@ def run_information(self):
         start_string += str(self.strategy_pack)
         return start_string
 
-    def auto_search(self, perc=1, status_update=None, max_time=None,
-                    save=False, smallest=False, genf=False):
+    def auto_search(self, **kwargs):
```

Comment:
Hiding all the argument in `kwargs` make it impossible for a user to know what he can pass as argument to the function without carefully reading the code of the function. I think the arguments should be detailed in the docstring.